.coveragerc

```diff
@@ -301,7 +301,7 @@ omit =
     homeassistant/components/freebox/switch.py
     homeassistant/components/fritz/device_tracker.py
     homeassistant/components/fritzbox_callmonitor/sensor.py
-    homeassistant/components/fritzbox_netmonitor/sensor.py
+    homeassistant/components/fritzbox_netmonitor/*
```

Comment:
This excludes the configuration flow from the tests. Please only exclude files that don't require testing.